components/dashboard/src/settings/Plans.tsx

```diff
@@ -294,8 +339,8 @@ export default function () {
     } else {
         const targetPlan = freePlan;
         let bottomLabel;
-        if (scheduledDowngradePlanId === targetPlan.chargebeeId) {
-            bottomLabel = <p className="text-green-600">Downgrade scheduled<br/><a className="text-blue-light leading-6" href="javascript:void(0)">Cancel</a></p>;
+        if (scheduledDowngradePlanId === targetPlan.chargebeeId && !pendingDowngradeCancellation) {
+            bottomLabel = <p className="text-green-600">Downgrade scheduled<br/><a className="text-blue-light leading-6" href="javascript:void(0)" onClick={() => confirmCancelDowngrade()}>Cancel</a></p>;
```

Comment:
**thought**: Wondering if there's a better copy for this one. Do you thinking the following could be better?

```suggestion
            bottomLabel = <p className="text-green-600">Downgrading ...<br/><a className="text-blue-light leading-6" href="javascript:void(0)" onClick={() => confirmCancelDowngrade()}>Cancel</a></p>;
```

Reply:
This takes until the next billing cycle, so the thought was:

- If I see `Downgrade scheduled`, I know everything is as expected and can continue with my life
- If I see `Downgrading ...`, the `...` tell me that I should stick around and wait for something to happen

I don't think the `...` is a good idea for things that can take several weeks. 😬

Reply:
Ah, ok! My bad! Ignore the comment in https://github.com/gitpod-io/gitpod/pull/3873#pullrequestreview-639052561, then! 😇